1.0/docs/devguide/templates.md

```diff
@@ -277,6 +283,23 @@ different name for the index property.
     </template>
     {% endraw %}
 
+### Forcing synchronous renders {#synchronous-renders}
+
+Call [`render`](/{% polymer_version_dir %}/api/#dom-repeat:method-render) 
+to force a `dom-repeat` template to synchronously render any changes to its 
+data. Normally changes are batched and rendered asynchronously. This is 
```

Comment:
Note that given the order of these sentences, "This is useful..." appears to refer to "Normally changes are batched..."

So probably, "This is useful in two scenarios:" => "Synchronous rendering has a performance cost, but can be useful in a few scenarios:"